src/KafkaNET.Library/Cfg/SyncProducerConfiguration.cs

```diff
@@ -35,7 +35,7 @@ public class SyncProducerConfiguration : ISyncProducerConfigShared
 
         public const int DefaultCorrelationId = -1;
 
-        public const string DefaultClientId = "";
+        public const string DefaultClientId = " ";
```

Comment:
@johnstark , Can we set it to something less obscure?
Java api does something like following which could help while debugging.

if (clientId.length() <= 0)
                clientId = "producer-" + PRODUCER_CLIENT_ID_SEQUENCE.getAndIncrement();